contents/docs/api/api.md

```diff
@@ -4,11 +4,15 @@ sidebar: Docs
 showTitle: true
 ---
 
-This section of our Docs explains how to pull or push data from/to our API.
+This section of our Docs explains how to pull or push data from/to our API. PostHog has an API available on all tiers of pricing and for every self-hosted version.
 
-If you're looking to push event or user data into PostHog instead of using a tracking snippet or a language-specific library, there is a simple guide in the [Dedicated Events API](/docs/integrations/api) section.
+Please note that PostHog makes use of two different APIs, serving different purposes and using different mechanisms for authentication. 
 
-PostHog has an API available on all tiers of pricing and for every self-hosted version.
+Our first API is used for pushing data into PostHog. This uses the 'Team API Key' that is included in the [frontend snippet](/docs/integration/js-integration). This API Key is **public**, and is what we use in our frontend integration to push events into PostHog, as well as to check for feature flags, for instance. 
```

Comment:
I'm not sure if ordering them "first" and "second" makes sense. Maybe something based on characteristics.

Reply:
Got it!